fastf1/events.py

```diff
@@ -981,16 +1002,21 @@ def get_event_by_name(
                 ``.get_event_by_name("british")`` and
                 ``.get_event_by_name("silverstone")`` will both return the
                 event for the British Grand Prix.
-            strict_search: Search only for exact query matches
+            strict_search: This argument is planned for deprecation. Use the
+                equivalent ``exact_match`` instead
+            exact_match: Search only for exact query matches
                 instead of using fuzzy search. For example,
                 ``.get_event_by_name("British Grand Prix",
-                strict_search=True)``
+                exact_match=True)``
                 will return the event for the British Grand Prix, whereas
-                ``.get_event_by_name("British", strict_search=True)``
+                ``.get_event_by_name("British", exact_match=True)``
                 will return ``None``
         """
-
         if strict_search:
+            warnings.warn(("This argument is planned for deprecation, "
```

Comment:
You should name the argument in the warning text to avoid confusion. "'strict_search' is deprecated, use... "

Reply:
Definitely, forgot these comments are not for docs